src/NzbDrone.Core/Notifications/CustomScript/CustomScript.cs

```diff
@@ -48,6 +48,7 @@ public override void OnGrab(GrabMessage message)
             environmentVariables.Add("Radarr_Movie_TmdbId", movie.TmdbId.ToString());
             environmentVariables.Add("Radarr_Movie_In_Cinemas_Date", movie.InCinemas.ToString() ?? string.Empty);
             environmentVariables.Add("Radarr_Movie_Physical_Release_Date", movie.PhysicalRelease.ToString() ?? string.Empty);
+            environmentVariables.Add("Radarr_Movie_Tag_Ids", string.Join("|", movie.Tags.Select(t => t.ToString()).ToArray()));
```

Comment:
Is the toarray needed? I think it'll work without.

Reply:
You are correct, it is not needed. Oversight on my part on other tests and leaving it there. Ive removed it and re-pushed it.